spec/datetimeformat.html

```diff
@@ -303,6 +306,16 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
             1. Else,
               1. Let _fv_ be an implementation and locale dependent String value representing `"ante meridiem"`.
             1. Add new part record { [[Type]]: `"dayPeriod"`, [[Value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal to `"millisecond"`, then
+            1. Let _msd_ be _dateTimeFormat_.[[MillisecondDigits]].
+            1. If _msd_ > 0, then
+              1. Let _v_ be _tm_.[[millisecond]].
+              1. If _msd_ = 1, then
+                1. Let _v_ be ? floor(_v_ / 100).
+              1. If _msd_ = 2, then
+                1. Let _v_ be ? floor(_v_ / 10).
+              1. Let _fs_ be the String consisting of the digits of the decimal representation of _v_.
```

Comment:
I think this part is still incorrect. I need to pad '0' for _fs_ to be length of _msd_. Any suggestion of how to write that?

Reply:
Should FormatNumber be called here? E.g., this could help take the numbering system into account.

Reply:
Good point. Let me try.